app/assets/stylesheets/custom.css.scss

```diff
@@ -226,3 +226,39 @@ h1 {
 .collection {
   padding: 3px;
 }
+
+.submit {
+  align: center;
+  float: left;
+  margin-left: 43%;
+  width: 75%
+}
+
+.form {
+  align: center;
+  float: left;
+  margin-left: 43%;
+  width: 75%
+}
+
+.order-dashboard-page {
+  align: center;
+  float: left;
+  margin-left: 23%;
+  width: 75%
+}
+
+.admin-links p {
+  border: 2px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered align, border, border-radius, margin-left, margin-right, padding, text-align